contrib/builder/rpm/centos-6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
```

Comment:
as well as which versions it is making rpms for

On Tue, May 5, 2015 at 3:49 PM, Jessica Frazelle <jess@docker.com> wrote:

> commited, well the deb ones are, its nice for visibility as to what is
> happening
>
> On Tue, May 5, 2015 at 3:49 PM, Vincent Batts <notifications@github.com>
> wrote:
>
>> In contrib/builder/rpm/centos-6/Dockerfile
>> <https://github.com/docker/docker/pull/12917#discussion_r29721453>:
>>
>> > @@ -0,0 +1,15 @@
>> > +#
>> > +# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
>>
>> then should these be committed? or .gitignore'd?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/pull/12917/files#r29721453>.
>>
>
>